sigopt/experiment_context.py

```diff
@@ -63,3 +64,16 @@ def get_runs(self):
 
   def get_best_runs(self):
     return self._connection.experiments(self.id).best_training_runs().fetch().iterate_pages()
+
+  def _parse_parameter(self, parameter):
+    if isinstance(parameter, Parameter):
+      parameter = parameter.as_json(parameter)
+      for attr in ['constraints', 'conditions']:
+        parameter.pop(attr, None)
```

Comment:
I'd rather leave these here and surface the api exception to the user. This will have the effect of removing the constraints and conditions I think (which is probably not allowed).

Reply:
This makes sense. Please remove them only if they are `None`

Reply:
If constraints and conditions are not `None`, can user still change bounds?

Reply:
perhaps in the future. and if we allowed it, then removing them here would have the effect of removing the constraints and conditions